.gitignore

```diff
@@ -184,7 +184,7 @@
 # ------------------------------------------------------------------------------
 
 # We want all custom site settings in settings.local.php
-!/web/sites/*/settings*.php
+/web/sites/*/settings*.php
```

Comment:
Shouldn't this be `/web/sites/*/settings.local.php` to ignore custom site settings and keep `web/sites/*/settings.php` under version control?

Reply:
Fixed